tests/test_moe_fp8_dnn.py

```
@@ -0,0 +1,124 @@
+        ([1, 3923, 374, 279, 6864, 315, 9822, 30], "What is the capital?"),
+        ([1, 8238, 264, 2875, 13, 5159], "Write a short function."),
+    ]
+
+    ok_count = 0
+    for i, (input_ids, prompt_text) in enumerate(test_cases):
+        gen_cfg = {
+            "top_k": 1,
+            "max_length": len(input_ids) + max_new_tokens,
+        }
+
+        status, handle, queue = engine.start_request(
```

Ruff (0.15.1)

[warning] Unpacked variable `status` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)